src/routes/contacts/+page.svelte

```diff
@@ -0,0 +1,74 @@
+<script>
+    import Graph from './Graph.svelte';
+    import Icon from 'svelte-icons-pack/Icon.svelte';
+    import BiMap from "svelte-icons-pack/bi/BiMap"; 
+
+    let windowWidth = 0;
+
+</script>
+
+<svelte:window bind:innerWidth={windowWidth} />
+
+<main class="flex flex-col justify-center my-20 -10 ">
+    <section class="flex flex-col text-white font-raleway text-center mb-4">
+        <h1 class="text-2xl font-bold">&lt; Contacta-nos /&gt;</h1>
+    </section>
+    <div class="md:grid grid-cols-2 mx-10">
+        <form class="justify-self-end min-w-[70%]">
+            <label class="m-1 text-white font-source-code" for="email">// Email</label><br>
+            <input class="rounded-lg p-1 text-primary placeholder-primary" type="email" id="email" placeholder="xxxxxxxxxxxxx@gmail.com"><br>
+            <label class="m-1 text-white font-source-code" for="name">// Nome</label><br>
+            <input class="rounded-lg p-1 text-primary placeholder-primary" type="text" id="name"><br>
+            <label class="m-1 text-white font-source-code" for="subject">// Assunto</label><br>
+            <input class="rounded-lg p-1 text-primary placeholder-primary" type="text" id="subject"><br>
+            <label class="m-1 text-white font-source-code" for="message">// Mensagem</label><br>
+            <textarea class="rounded-lg p-1 text-primary placeholder-primary" id="message"></textarea><br>
```

Comment:
- Maybe each label/input pair could be converted into a component? I imagine we could potentially have inputs like these in other places of the website, plus it improves readability and reduces code repetition. The parameters to be passed could be the `for`, `type`and `placeholder` labels.
- The font is actually named `source_code`and not `source-code`, I changed it in my commit to avoid putting brackets around the name in tailwind config file. In hindsight, that might not have been the best decision, as it looks strange in a tailwind class. Maybe it could be reverted and you don't need to change the name here, but just know that the font is not being rendered properly with this code. We'll discuss that in the meeting.